interfaces/cython/cantera/test/test_convert.py

```diff
@@ -717,3 +718,450 @@ def test_ch4_ion(self):
         self.checkThermo(ctiGas, yamlGas, [300, 500, 1300, 2000])
         self.checkKinetics(ctiGas, yamlGas, [900, 1800], [2e5, 20e5])
         self.checkTransport(ctiGas, yamlGas, [298, 1001, 2400])
+
+class ctml2yamlTest(utilities.CanteraTest):
+
+    def checkConversion(self, basename, cls=ct.Solution, ctmlphases=(),
+                        yamlphases=(), **kwargs):
+        ctmlPhase = cls(basename + '.xml', adjacent=ctmlphases, **kwargs)
+        yamlPhase = cls(basename + '.yaml', adjacent=yamlphases, **kwargs)
+
+        self.assertEqual(ctmlPhase.element_names, yamlPhase.element_names)
+        self.assertEqual(ctmlPhase.species_names, yamlPhase.species_names)
+        self.assertEqual(ctmlPhase.n_reactions, yamlPhase.n_reactions)
+        for C, Y in zip(ctmlPhase.species(), yamlPhase.species()):
+            self.assertEqual(C.composition, Y.composition)
+
+        for i, (C, Y) in enumerate(zip(ctmlPhase.reactions(),
+                                       yamlPhase.reactions())):
+            self.assertEqual(C.__class__, Y.__class__)
+            self.assertEqual(C.reactants, Y.reactants)
+            self.assertEqual(C.products, Y.products)
+            self.assertEqual(C.duplicate, Y.duplicate)
```

Comment:
The enumeration here is unused.

Reply:
I copied this implementation from `cti2yamlTest`, so I removed it there too. I also removed the class setup for `cti2yamlTest`, and moved the conversion of `gri30` into the `test_gri30` method.